pkg/schedule/schedulers/evict_slow_trend.go

```diff
@@ -350,7 +413,7 @@ func checkStoresAreUpdated(cluster sche.SchedulerCluster, slowStoreID uint64, sl
 			updatedStores += 1
 			continue
 		}
-		if slowStoreRecordTS.Before(store.GetLastHeartbeatTS()) {
+		if slowStoreRecordTS.Compare(store.GetLastHeartbeatTS()) <= 0 {
```

Comment:
Why change it?

Reply:
`Before` is not accurate for the judgement so I changed it.

When `slowStore.LastHeartBeatTS == store.LastHeartBeatTS`, it might miss it. However, in this case, it means that the PD simultaneously receives the heartbeats from slow node and other nodes. And it also means the heartbeat is already fresh to make the judgement, no need to wait another heartbeat tick.